<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="sub_topic_name" default="/sensing/gnss/nmea_sentence"/>
  <arg name="pub_fix_topic_name" default="/fix"/>
  <arg name="pub_gga_topic_name" default="/gga"/>
  <arg name="output_gga" default="true"/>

  <node pkg="nmea2fix" name="nmea2fix" exec="nmea2fix">
    <param name="sub_topic_name" value="$(var sub_topic_name)"/>
    <param name="pub_fix_topic_name" value="$(var pub_fix_topic_name)"/>
    <param name="pub_gga_topic_name" value="$(var pub_gga_topic_name)"/>
    <param name="pub_gga_topic_name" value="$(var pub_gga_topic_name)"/>
    <param name="output_gga" value="$(var output_gga)"/>
  </node>

</launch>
